web_programming/crawl_google_results.py

```diff
@@ -1,24 +1,32 @@
-import sys
 import webbrowser
+from sys import argv
+from urllib.parse import quote
 
-import requests
 from bs4 import BeautifulSoup
-from fake_useragent import UserAgent
+from requests import get
```

Comment:
```suggestion
import requests
```

A function called `get()` is not self-documenting and may confuse the reader ("Get what from where?") so lets use `requests.get()` instead.

Reply:
ok sure!

Reply:
~~from requests import get~~ `import requests`
**Changed!**